x/dex/keeper/pool.go

```diff
@@ -85,7 +118,6 @@ func (k Keeper) GetPool(
 	case lowerTickFound && !upperTickFound:
 		upperTick = types.NewPoolReservesFromCounterpart(lowerTick)
 	case !lowerTickFound && !upperTickFound:
-		// Pool has already been initialized before so we can safely assume that pool creation doesn't throw an error
```

Comment:
Maybe just personal preference, but I like comments explaining why we chose to use a method that panics. Otherwise, it can look like a mistake to someone later.